package.json

```diff
@@ -17,6 +17,7 @@
     "test": "jest"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-free": "^5.15.3",
```

Comment:
Unfortunately it needs the other fontawesome packages :( 

    "@fortawesome/fontawesome-svg-core": "^1.2.35",
    "@fortawesome/react-fontawesome": "^0.1.14",